CHANGELOG.md

```diff
@@ -6,7 +6,32 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
-[Unreleased]: https://github.com/bottlerocket-os/bottlerocket-test-system/compare/v0.0.15...develop
+[Unreleased]: https://github.com/bottlerocket-os/bottlerocket-test-system/compare/v0.0.16...develop
+
+## [0.0.16] - 2025-03-12
+
+### Fixed
+
+- fix NVIDIA workload test ([#979])
+
+### Added
+
+- workload: add tests for ruby and dotnet ([#972])
+- vSphere-k8s-cluster-agent: Add licenseToken to cluster configurations ([#975])
+- ec2 agent: Implement EBS volume provisioning support ([#978])
+
+### Changed
+
+- ec2-resource-agent: Remove ecr-credential-provider from userdata ([#973])
+- eks-resource-agent: default ManagedNodeGroup desiredCapacity to 0 ([#974])
+
```

Comment:
We should include this PR as well. https://github.com/bottlerocket-os/bottlerocket-test-system/pull/959